keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ApplicationLoadBalancerHandler.kt

```diff
@@ -0,0 +1,211 @@
+package com.netflix.spinnaker.keel.ec2.resource
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.keel.api.Resource
+import com.netflix.spinnaker.keel.api.ResourceKind
+import com.netflix.spinnaker.keel.api.ResourceName
+import com.netflix.spinnaker.keel.api.SPINNAKER_API_V1
+import com.netflix.spinnaker.keel.api.ec2.ApplicationLoadBalancer
+import com.netflix.spinnaker.keel.api.ec2.LoadBalancerType
+import com.netflix.spinnaker.keel.api.ec2.cluster.Location
+import com.netflix.spinnaker.keel.clouddriver.CloudDriverCache
+import com.netflix.spinnaker.keel.clouddriver.CloudDriverService
+import com.netflix.spinnaker.keel.diff.ResourceDiff
+import com.netflix.spinnaker.keel.ec2.CLOUD_PROVIDER
+import com.netflix.spinnaker.keel.events.TaskRef
+import com.netflix.spinnaker.keel.model.Job
+import com.netflix.spinnaker.keel.model.Moniker
+import com.netflix.spinnaker.keel.model.OrchestrationRequest
+import com.netflix.spinnaker.keel.model.OrchestrationTrigger
+import com.netflix.spinnaker.keel.orca.OrcaService
+import com.netflix.spinnaker.keel.plugin.ResourceHandler
+import com.netflix.spinnaker.keel.plugin.ResourceNormalizer
+import com.netflix.spinnaker.keel.retrofit.isNotFound
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import retrofit2.HttpException
+
+class ApplicationLoadBalancerHandler(
+  private val cloudDriverService: CloudDriverService,
+  private val cloudDriverCache: CloudDriverCache,
+  private val orcaService: OrcaService,
+  override val objectMapper: ObjectMapper,
+  override val normalizers: List<ResourceNormalizer<*>>
+) : ResourceHandler<ApplicationLoadBalancer> {
+  override val log: Logger by lazy { LoggerFactory.getLogger(javaClass) }
+
+  override val apiVersion = SPINNAKER_API_V1.subApi("ec2")
+  override val supportedKind = ResourceKind(
+    apiVersion.group,
+    "alb",
+    "albs"
```

Comment:
Pedantic but would it make sense to use something more verbose here? We currently have `cluster`, `security-group` and `image`. Is `application-load-balancer` too long? I like the clarity myself.

Reply:
I like keeping it pithy but I'm fine with verbose, will update `clb` to `classic-load-balancer` as well.